pages/art-pieces/[slug].js

```diff
@@ -0,0 +1,29 @@
+import ArtPieceDetails from "../../components/ArtPieceDetails";
+import { useRouter } from "next/router";
+
+export default function ArtPieceDetailsPage({ data }) {
+  const router = useRouter();
+  const { slug } = router.query;
+  console.log(slug);
+  const findImage = data.find((piece) => {
+    return piece.slug === slug;
+  });
+  const { imageSource, name, artist, year, genre } = findImage;
+  console.log(findImage.name);
+  return (
+    <>
+      <button type="button" onClick={() => router.push("/art-pieces")}>
+        🔙 Art Gallery
+      </button>
```

Comment:
Since you don't do any calculations in the onClick function you should better use a ``<Link>`` instead.